csharp/sample/Microsoft.ML.OnnxRuntime.ResNet50v2Sample/README.md

```diff
@@ -0,0 +1,169 @@
+# C# Sample: ResNet50 v2
+
```

Comment:
Thank you for such a clear and detailed readme!

Reply:
No worries! 😊